src/cudamatrix/cu-array-inl.h

```diff
@@ -212,7 +212,8 @@ void CuArrayBase<T>::SetZero() {
 #if HAVE_CUDA == 1
   if (CuDevice::Instantiate().Enabled()) {
     CuTimer tim;
-    CU_SAFE_CALL(cudaMemset(this->data_, 0, this->dim_ * sizeof(T)));
+    CU_SAFE_CALL(cudaMemsetAsync(this->data_, 0, this->dim_ * sizeof(T),cudaStreamPerThread));
```

Comment:
space after comma, please (google style guide)-- and line break if going over 80 chars per line.

Reply:
should be fixed now